docs/runbooks/neon-cost-profiling.md

```
@@ -0,0 +1,201 @@
+WHERE relname IN ('catalog_items', 'pack_items', 'packs')     -- swap in tables you care about
+ORDER BY seq_tup_read DESC;
+```
+
+What the columns tell you:
+
+- `seq_tup_read` deltas — the projection wins land here. If U2-U6 cut catalog SELECTs in half, this drops in half.
+- `n_tup_upd` divided by row count — a "lifetime updates per row" sanity check. Anything above ~5 is suspicious and worth investigating (the catalog-projections audit surfaced ~44 updates/row, which traced back to a no-op embedding-regen diff loop).
+- `idx_scan` vs `seq_scan` ratio — on small tables (< ~1 MB) Postgres prefers seq scan and that's correct; on large tables a high seq_scan count usually means a missing or unused index.
+
+To reset and get a clean window (requires owner role):
+```sql
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)